go/nomdl/parser.go

```diff
@@ -100,7 +100,7 @@ func (p *Parser) ensureAtEnd() {
 //   Ident
 //
 // StructField :
-//   StructFieldName `:` Type
+//   StructFieldName `?`? `:` Type
```

Comment:
"Yo, dawg, I put a question mark by your question mark so that your optional marker can be optional"